APITest/APITest/Models/EmployeesModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace APITest.Models
+{
+
+    public class EmployeesModel
+    {
+        public string status { get; set; }
+        public Datum[] data { get; set; }
+    }
+
+    public class Datum
+    {
+        public uint id { get; set; }
+        public string employee_name { get; set; }
+        public uint employee_salary { get; set; }
+        public byte employee_age { get; set; }
+        public object profile_image { get; set; }
```

Comment:
Data duplication. You have such data in the Data class

Reply:
EmployeeModel and EmployeesModel are containing the different types of Data class: a set of fields and an array of sets of fields